cookiecutter.json

```diff
@@ -1,5 +1,8 @@
 {
     "project_name": "restplus_api",
     "app_name": "myapi",
+    "flask-restplus":"n",
```

Comment:
How would application work without flask-restplus? It will break all tests
Without gunicorn, how would we be able to deploy application?